src/lib/journald.ts

```diff
@@ -10,7 +10,8 @@ export function spawnJournalctl(opts: {
 	containerId?: string;
 	format: string;
 	filterString?: string;
-	since?: number;
+	since?: string;
+	until?: string;
```

Comment:
```suggestion
	since?: number;
	until?: number;
```
@rkeulemans , I believe that would be because of the type change from number to string - you should be able to fix it by change the types of `since` and `until` back to number. The `args.join(' ')` call near the end of `spawnJournalCtl` will normalize the types. The specific error is because the `backfill` method in `src/logging/monitor.ts` passes a number type to as an opt for `since`.